packages/core/http/core-http-router-server-internal/src/response.test.ts

```diff
@@ -6,13 +6,79 @@
  * Side Public License, v 1.
  */
 
-import { fileResponseFactory } from './response';
+import { kibanaResponseFactory } from './response';
+
+describe('kibanaResponseFactory', () => {
+  describe('status codes', () => {
+    const tests = [
+      { name: 'ok', expectedStatusCode: 200, response: kibanaResponseFactory.ok() },
+      { name: 'created', expectedStatusCode: 201, response: kibanaResponseFactory.created() },
+      { name: 'accepted', expectedStatusCode: 202, response: kibanaResponseFactory.accepted() },
+      { name: 'noContent', expectedStatusCode: 204, response: kibanaResponseFactory.noContent() },
+      {
+        name: 'multiStatus',
+        expectedStatusCode: 207,
+        response: kibanaResponseFactory.multiStatus(),
+      },
+      {
+        name: 'redirected',
+        expectedStatusCode: 302,
+        response: kibanaResponseFactory.redirected({}),
+      },
+      {
+        name: 'notModified',
+        expectedStatusCode: 304,
+        response: kibanaResponseFactory.notModified({}),
+      },
+      { name: 'badRequest', expectedStatusCode: 400, response: kibanaResponseFactory.badRequest() },
+      {
+        name: 'unauthorized',
+        expectedStatusCode: 401,
+        response: kibanaResponseFactory.unauthorized(),
+      },
+      { name: 'forbidden', expectedStatusCode: 403, response: kibanaResponseFactory.forbidden() },
+      { name: 'notFound', expectedStatusCode: 404, response: kibanaResponseFactory.notFound() },
+      { name: 'conflict', expectedStatusCode: 409, response: kibanaResponseFactory.conflict() },
+      {
+        name: 'unprocessableContent',
+        expectedStatusCode: 422,
+        response: kibanaResponseFactory.unprocessableContent(),
+      },
+      {
+        name: 'file',
+        expectedStatusCode: 200,
+        response: kibanaResponseFactory.file({
+          filename: 'test.txt',
+          body: 'content',
+        }),
+      },
+      {
+        name: 'custom 205',
+        expectedStatusCode: 205,
+        response: kibanaResponseFactory.custom({
+          statusCode: 205,
+        }),
+      },
+      {
+        name: 'customError 505',
+        expectedStatusCode: 505,
+        response: kibanaResponseFactory.customError({
+          statusCode: 505,
+        }),
+      },
+    ];
+
+    tests.forEach(({ name, expectedStatusCode, response }) => {
```

Comment:
FWIW jest does support [table driven tests](https://jestjs.io/docs/api#testeachtablename-fn-timeout)

Reply:
Neat, didn't know, I'll rewrite to use that instead!

Reply:
Fixed in https://github.com/elastic/kibana/pull/186379/commits/fbb3d6e6a2a8c371baff4faf37e95703d0baa09a